cmd/keeper/cmd/keeper.go

```diff
@@ -558,7 +558,6 @@ func (p *PostgresKeeper) updatePGState(pctx context.Context) {
 	pgState, err := p.GetPGState(pctx)
 	if err != nil {
 		log.Errorw("failed to get pg state", zap.Error(err))
-		return
```

Comment:
For the other comment just remote the error check since GetPGState will become:
```
func (p *PostgresKeeper) GetPGState(pctx context.Context) *cluster.PostgresState
```